recipes/gst-plugins-bad-1.0/0005-amcvideodec-add-property-to-control-frame-droppin.patch

```diff
@@ -0,0 +1,131 @@
+From 36964d37a5142044a61993ac0f5cd8ccd9404bca Mon Sep 17 00:00:00 2001
+From: Mark Fernie <39318992+drmarkfernie@users.noreply.github.com>
+Date: Fri, 10 Dec 2021 16:20:09 +1100
+Subject: [PATCH] amcvideodec: add property to control frame droppin
+
+---
+ sys/androidmedia/gstamcvideodec.c | 60 +++++++++++++++++++++++++++++--
+ sys/androidmedia/gstamcvideodec.h |  4 +++
+ 2 files changed, 62 insertions(+), 2 deletions(-)
+
+diff --git a/sys/androidmedia/gstamcvideodec.c b/sys/androidmedia/gstamcvideodec.c
+index d880c58..7e46c2e 100644
+--- a/sys/androidmedia/gstamcvideodec.c
++++ b/sys/androidmedia/gstamcvideodec.c
+@@ -255,9 +255,13 @@ static void
+ gst_amc_video_dec_on_frame_available (JNIEnv * env, jobject thiz,
+     long long context, jobject surfaceTexture);
+ 
++/* properties */
++#define DEFAULT_DROP_FRAMES       TRUE
++
+ enum
+ {
+-  PROP_0
++  PROP_0,
++  PROP_DROP_FRAMES,
+ };
+ 
+ /* class initialization */
+@@ -338,6 +342,38 @@ caps_to_mime (GstCaps * caps)
+   return NULL;
+ }
+ 
++static void
++gst_amc_video_dec_get_property (GObject * object, guint property_id,
++    GValue * value, GParamSpec * pspec)
++{
++  GstAmcVideoDec *decoder = GST_AMC_VIDEO_DEC (object);
++
++  switch (property_id) {
++    case PROP_DROP_FRAMES:
++      g_value_set_boolean (value, decoder->drop_frames);
```

Comment:
Shouldn't this be `get_boolean`, and vice versa?

Reply:
No, it is correct

Reply:
it's "setting" the returned value (which is a parameter)